decidim-comments/app/frontend/comments/comment.fragment.graphql

```diff
@@ -1,12 +1,12 @@
 fragment Comment on Comment {
   ...CommentData
-  replies {
+  comments {
```

Comment:
I agree. I think we can use another approach in order to allow infinite nesting (as long as the ui supports it).